### ISSUE TITLE
Popovers - Need a close option

### ISSUE THREAD

**Issue by bmrgould** [gctools-outilsgc/gccollab-mobile#190]
Normal way to exit out of it is to click off the popover on the grayed out original page.

For google TalkBack tabbing/swiping through content, cannot target outside of the popover to close it. Add a physical real close option.
